apps/chat-e2e/src/tests/sharedFilesAttachments.test.ts

```diff
@@ -979,11 +998,46 @@ dialSharedWithMeTest(
       },
     );
 
+    await dialSharedWithMeTest.step(
```

Comment:
maybe, we could speed-up the step using API instead of UI actions?

Reply:
done